src/main/java/pl/edu/pw/mini/ingreedio/api/controller/ProductController.java

```diff
@@ -29,28 +35,41 @@
 @Tag(name = "Products" /*, description = "..."*/)
 public class ProductController {
     private final ProductService productService;
+    private final ProductDtoMapper productDtoMapper;
+    private final int pageSize = 10;
```

Comment:
I think that it would be nice to have the `pageSize` configurable from the `application.properties`. See the `JwtService` for example:

![image](https://github.com/Java-DZGS/ingreedio-api/assets/56317134/aa9c21dc-9e02-4491-8ace-90faef142269)

and the `application.properties`:

![image](https://github.com/Java-DZGS/ingreedio-api/assets/56317134/748c540b-7d73-40d4-8957-10f81c608e46)

Remember to define these variables in both -- `test/*/application.properties` and `main/*/application.properties`.

Also I believe that such a variable should be in service, I think that the controller should be responsible for running a proper service only.

I think that it would be better to move the logic of the `public ResponseEntity<ProductListResponseDto> getProducts` to the service (see the next comment).